dist/palindrom-dom.js

```diff
@@ -2410,7 +2408,7 @@ var Palindrom = (function () {
     var error = this.jsonpatch.validate(sequence, tree);
     if (error) {
       error.message = "Outgoing patch validation error: " + error.message;
-      dispatchErrorEvent(this, error);
+      this.onGenericError(error);
```

Comment:
I think that it would be better to split it to: `OnIncomingPatchValidationError`, `OnOutgoingPatchValidationError` and remove the line that assigns `error.message =`